grype/pkg/syft_provider.go

```diff
@@ -11,7 +11,7 @@ func syftProvider(userInput string, config ProviderConfig) ([]Package, Context,
 		return nil, Context{}, nil, errDoesNotProvide
 	}
 
-	sourceInput, err := source.ParseInput(userInput, config.Platform)
+	sourceInput, err := source.ParseInputWithName(userInput, config.Platform, "", config.DefaultImagePullSource)
```

Comment:
is the name supposed to be included here instead of `""`?

Reply:
Ohh no! Good catch - dropped that commit earlier to keep them small and didn't add it back in